typings/number_format.d.ts

```diff
@@ -39,7 +39,7 @@ declare module "react-number-format" {
     removeFormatting?: (formattedValue: string) => string;
     mask?: string | string[];
     value?: number | string;
-    defaultValue?: number | string;
+    defaultValue?: number | string | string[] | readonly string[];

```

Comment:
But isn't this wrong.?  defaultValue will never be array of string.

Reply:
@s-yadav 
I was thinking the same, but apparently it can literally take `any`.
See here: https://codesandbox.io/s/awesome-benz-7vj06 with different `defaultValue` types

For example, an array of string such as:
`const myArray = ["first", "second", "third"]`
if assigned to `defaultValue` will produce:
`first, second, third`

Reply:
It can because of typecasting by JS. But that will still be wrong. Also, the type of value and default value should match.
Wrong typing can lead to confusion on dev end.

Is there any other way to force type in material ui.?

I guess the Material-UI has mentioned it as any because the Material UI supports the text field to be rendered as select.